environment.yml

```diff
@@ -22,7 +22,15 @@ dependencies:
   - conda-forge::matplotlib=>3.6.0
   - conda-forge::keras-tuner=>1.1.3
   - conda-forge::hiplot=>0.1.33
+  - conda-forge::optuna
+  - conda-forge::pytorch=>2.4
+  - conda-forge::torchvision
+  - conda-forge::plotly
+  - conda-forge::pyarrow
+  - conda-forge::lxml
+
   - pip:
       - blitzgsea==1.3.33
+      
```

Comment:
Do these packages NEED to be these versions? or would another suffice? Consider keeping environemnt more open. Up to you though!

Reply:
pytorch does have to be that version - anything else that has a version number is originally from the environment that came before me so too many changes seem to mess things up